tracing-subscriber/src/fmt/format/json.rs

```diff
@@ -209,7 +209,10 @@ where
 
             let current_span = if self.format.display_current_span || self.format.display_span_list
             {
-                ctx.ctx.current_span().id().and_then(|id| ctx.ctx.span(id))
+                event
+                    .parent()
+                    .and_then(|id| ctx.ctx.span(id))
```

Comment:
since [`FmtContext` has a `span` method](https://docs.rs/tracing-subscriber/0.2.17/tracing_subscriber/fmt/struct.FmtContext.html#implementations) (which I think it didn't have when this code was originally written) we can remove the `ctx.ctx`:
```suggestion
                    .and_then(|id| ctx.span(id))
```